sdk/nodejs/cmd/run-policy-pack/run.ts

```diff
@@ -18,8 +18,7 @@ import * as path from "path";
 import * as tsnode from "ts-node";
 import { ResourceError, RunError } from "../../errors";
 import * as log from "../../log";
-import { disconnectSync } from "../../runtime/settings";
-import { runInPulumiStack } from "../../runtime/stack";
+import * as runtime from "../../runtime";
```

Comment:
This change [matches the imports](https://github.com/pulumi/pulumi/blob/7b3ec744f467f0ecea7d3ab8cfc0df6e39dbd9f1/sdk/nodejs/cmd/run/run.ts#L21) used when running normal Node.js-based Pulumi programs, so that the modules are loaded in the same order. Otherwise, `getProject` and `getStack` do not work inside Policy Packs, because of circular module dependencies. If you're curious, more details [here](https://github.com/pulumi/pulumi-policy/issues/78#issuecomment-562203710).